pandas/core/arrays/sparse/array.py

```diff
@@ -452,7 +452,7 @@ def from_spmatrix(cls, data):
 
         return cls._simple_new(arr, index, dtype)
 
-    def __array__(self, dtype=None, copy=True) -> np.ndarray:
+    def __array__(self, dtype=None) -> np.ndarray:
```

Comment:
i know we removed something similar this in a recent PR, but are we a) actually hitting this and b) is this being passed at all?

Reply:
I don't think so, that's why I removed it. Although it will fail CI since some tests use it. I will have a better look at this en comment the results here.

Reply:
revert this change